Makefile

```diff
@@ -866,15 +863,17 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+LINT_TARGETS = benchmark doc lib test tools
```

Comment:
Nit: maybe JSLINT_TARGETS since these are the directories for JavaScript linting but there's a separate C++ linting process that lints stuff elsewhere?